Temporal Cloud limits sheet - remove KB (#2262)
temporalio/documentation: docs-src/references/dynamic-configuration.md

The default values on these keys are based on extensive testing.

> You can change these values, but ensure that you are provisioning enough database resources to handle the changed values.

Reviewer: ```suggestion
When changing these values, make sure that you're provisioning enough database resources to handle these changes.
```
